### PR TITLE
 Export consignment as BagIt bag

### PR DESCRIPTION
**Pull request by suzannehamilton**
BagIt is a specification created by the US Library of Congress, and is used to store and describe arbitrary collections of digital content.

It is a common archival format, so exporting a consignment as a Bag is a flexible way for us to export the data, as opposed to saving the data in the specific way required by our preservation system. We will have to convert the BagIt data to a preservation SIP (Submission Information Package) but we expect that to be straightforward.

Save a consignment as a BagIt bag which contains the consignment's files and adds metadata about the consignment and the individual files, e.g. the original client-side checksum.

For more information about BagIt, see:

- https://tools.ietf.org/html/rfc8493
- https://blogs.loc.gov/thesignal/2019/04/bagit-at-the-library-of-congress/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,28 +9,34 @@ Digital Archivist might use to export files from S3 once a transfer has been fin
...
 - Zip the files
-- Encrypt the zip file
 - Upload the encrypted file to a different S3 bucket
 
 You can run the steps separately, or run them together with Docker.
 
-### Step 1: download files
+### Step 1: download and package files
 
 [Configure your AWS credentials][aws-cli-auth] in the `~/.aws/credentials` file. The download step will uses this
```
Comment (TomJKing) [inline README.md]: Should read: "The download step will use this..."

File: README.md [hunk 2]
```diff
@@ -57,12 +63,15 @@ setting the `ARCHIVE_FILEPATH` variable to the file to be uploaded.
...
 - Build the image with `docker build . --tag exportfiles`
-- Run the Docker image, setting environment variables with your AWS key ID and AWS secret key:
+- Run the Docker image, setting environment variables:
 
   ```
   docker run \
     --env ACCESS_KEY_ID=your_aws_key_id \
     --env SECRET_ACCESS_KEY=your_aws_secret_key \
+    --env GRAPHQL_SERVER=https://graphql-api-hostname.amazonaws.com \
+    --env GRAPHQL_SERVER=some/api/path \
```
Comment (TomJKing) [inline README.md]: Should be: --env GRAPHQL_PATH=some/api/path

**Review by TomJKing — APPROVED** [review]
Couple of minor comments

File: README.md (hunk 1 above)
Comment (suzannehamilton) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (suzannehamilton) [reply README.md]: Oops. :woman_facepalming: Fixed.

**Pull request merged**